core/src/main/scala/org/apache/spark/MapOutputTracker.scala

```diff
@@ -335,23 +335,6 @@ private[spark] abstract class MapOutputTracker(conf: SparkConf) extends Logging
    *         tuples describing the shuffle blocks that are stored at that block manager.
    */
   def getMapSizesByExecutorId(
-      shuffleId: Int,
-      startPartition: Int,
-      endPartition: Int)
-  : Iterator[(BlockManagerId, Seq[(BlockId, Long, Int)])]
-
-  /**
-   * Called from executors to get the server URIs and output sizes for each shuffle block that
-   * needs to be read from a given range of map output partitions (startPartition is included but
-   * endPartition is excluded from the range) and is produced by
-   * a range of mappers (startMapIndex, endMapIndex, startMapIndex is included and
-   * the endMapIndex is excluded).
```

Comment:
Hi, @Ngone51 . This should be the function description of the unified `getMapSizesByExecutorId`.

Reply:
Specifically, `endMapIndex == Int.MaxValue` seems worth to mention in a way here.

Reply:
Yea, updated, thanks!